cmd/lookout/event.go

```diff
@@ -93,6 +94,10 @@ func (c *EventCommand) makeDataServerHandler() (*lookout.DataServerHandler, erro
 	bblfshConn, err := grpc.DialContext(timeoutCtx, c.Bblfshd, grpc.WithInsecure(), grpc.WithBlock())
 	if err != nil {
 		log.Warningf("bblfsh service is unavailable. No UAST will be provided to analyzer. Error: %s", err)
```

Comment:
While here, why not improving this error msg as well?

`"bblfshd instance could not be found at %s. No UASTs will be available to analyzers. Error: %s"`